torch/_inductor/ir.py

```diff
@@ -7515,6 +7519,17 @@ def create_output(output: IRNode, ind: int):
                     skip_size_stride_alignment_checks=True,
                 )
 
+        # Force the output strides to be same as the original strides
```

Comment:
This needs a test at the very least. You can add an invoke_subgraph node, then do a graph pass that changes the outputs in the invoke_subgraph subgraph, and then check to make sure the strides are still what you expect.

Reply:
Yes, I was not able to get a test working.

I was looking at a regression when I wrap the whole model with the invoke subgraph. When I diffed the output code, I saw an extra kernel after the invoke subgraph call, even though there was no operation outside of the invoke subgraph call. So this PR was my attempt to make the stride of invoke subgraph same as eager output to avoid that extra kernel. But after your comment about passes changing meta vals, I am not sure if this is correct (or what should be the solution to avoid the extra kernel)